src/Form/test-cases/chewy_checkout.html

```diff
@@ -1012,7 +1012,7 @@ <h2 class="kib-tile__title">
                     data-qa-id="place-order-button" data-event-category="checkout" data-event-action="impression"
```

Comment:
This form also has a "ship to this address" button that could be considered a submit button:

<img width="1440" src="https://user-images.githubusercontent.com/90369497/158899569-0b572f71-ee8d-48b2-8626-9a86ba06144f.png">